x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -94,6 +94,10 @@ const getArtifactIds = (manifest: ManifestSchema) =>
 const manifestsEqual = (manifest1: ManifestSchema, manifest2: ManifestSchema) =>
   isEqual(new Set(getArtifactIds(manifest1)), new Set(getArtifactIds(manifest2)));
 
+/**
+ * Manages the Endpoint Integration Policy manifest that contains information about Kibana user generated Artifacts that
+  * are used by the Elastic Endpoint. Examples of artifacts include Exception Lists, Trusted Application and Event Filters.
```

Comment:
Needs one less space here for eslint. I'd also maybe mention that this provides a central place to perform synchronous operations on the artifacts... unless this has changed. @paul-tavares @oatkiller